sys-fs/btrfsmaintenance/btrfsmaintenance-0.3.1.ebuild

```diff
@@ -0,0 +1,52 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+inherit eutils
+
+DESCRIPTION="Scripts for btrfs maintenance tasks like periodic scrub, balance, trim or defrag"
+HOMEPAGE="https://github.com/kdave/btrfsmaintenance"
+GITTAG="v${PV}"
```

Comment:
Why the extra variable? It's used only once, and it doesn't look like it's likely to change.